simtools/db_handler.py

```diff
@@ -174,7 +176,35 @@ def getModelParameters(
                 onlyApplicable,
             )
 
+    def getFileDB(self, dbName, dest, fileName):
+        """
+        Get a file from the DB and write it to disk.
+
+        Parameters
+        ----------
+        dbName: str
+            Name of the DB to search in.
+        dest: str or Path
+            Location where to write the file to.
+        fileName: str
+            Name of the file to get.
+        """
+
+        self._logger.debug(f"Getting {fileName} and writing it to {dest}")
+        file = self._getFileMongoDB(dbName, fileName)
+        self._writeFileFromMongoToDisk(dbName, dest, file)
+
     def exportModelFiles(self, parameters, dest):
```

Comment:
I comment on the different naming:
- `getFileDB` is getting something from the DB and writing to disk
- `exportModelFiles` is getting something from the DB and writing to disk

Reply:
Yes, this is the result of two people with two different opinions =)
Which name would you prefer? You will be the tie break.

Reply:
Whatever - the opinion of the active developer counts.

Reply:
Fixed.